### PR TITLE
Add w3lib and czml3

### PR DESCRIPTION
**Pull request by Juanlu001**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/czml3/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True
+
+requirements:
+  host:
+    - python>=3.6
```
Comment (chrisburr) [inline recipes/czml3/meta.yaml]: You need a space here:
```suggestion
    - python >=3.6
```

File: recipes/czml3/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+    - pip
+  run:
+    - python>=3.6
```
Comment (chrisburr) [inline recipes/czml3/meta.yaml]:
```suggestion
    - python >=3.6
```

File: recipes/czml3/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    - python-dateutil>=2.7,<3
```
Comment (chrisburr) [inline recipes/czml3/meta.yaml]:
```suggestion
    - python-dateutil >=2.7,<3
```

File: recipes/czml3/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: d97f0b503de13faeb5e8dab7165279b2f1d948286b60378afe6e8dcef5a70df3
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True
```
Comment (chrisburr) [inline recipes/czml3/meta.yaml]: Why is this here?
```suggestion
```

Comment (Juanlu001) [reply recipes/czml3/meta.yaml]: Oops, leftover from a wrong version of applying the version specifiers

**Comment by Juanlu001** [comment]
Oops! This will need a rebase :) Thanks bot! 🤖

**Comment by Juanlu001** [comment]
@conda-forge-admin, please ping team
